Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -64,9 +68,26 @@ def run
         Homebrew.with_no_api_env do
           eval_all = args.eval_all? || Homebrew::EnvConfig.eval_all?
 
-          formulae_and_casks = if args.tap
+          formulae_and_casks = if args.auto?
+            raise UsageError, "`--formula` or `--cask` must be passed with `--auto`." if !args.formula? && !args.cask?
+
+            tap_arg = args.tap
+            raise UsageError, "`--tap=` must be passed with `--auto`." if tap_arg.blank?
+
+            tap = Tap.fetch(tap_arg)
+            autobump_list = tap.autobump
+            what = args.cask? ? "casks" : "formulae"
+            raise UsageError, "No autobumped #{what} found." if autobump_list.blank?
+
+            autobump_list.map do |name|
+              qualified_name = "#{tap.name}/#{name}"
+              next Cask::CaskLoader.load(qualified_name) if args.cask?
+
+              Formulary.factory(qualified_name)
+            end
+          elsif args.tap
             tap = Tap.fetch(T.must(args.tap))
-            raise UsageError, "`--tap` cannot be used with official taps." if tap.official?
+            raise UsageError, "`--tap` without `--auto` cannot be used with official taps." if tap.official?
```

Comment:
```suggestion
            raise UsageError, "`--tap` can only be used with `--auto` for official taps." if tap.official?
```

Let's get rid of the double negation here.